clang/test/OpenMP/depobj_codegen.cpp

```diff
@@ -36,9 +45,24 @@ int main(int argc, char **argv) {
 #pragma omp depobj(b) update(mutexinoutset)
 #pragma omp depobj(a) depend(iterator(char *p = argv[argc]:argv[0]:-1), out: p[0])
   (void)tmain(a), tmain(b);
+ tmainc();
```

Comment:
here as well, at least not making it worse

Reply:
same !!

Reply:
ping @shiltian updated !!